src/devices/bus/coco/coco_midi.cpp

```diff
@@ -0,0 +1,130 @@
+// license:BSD-3-Clause
+// copyright-holders:tim lindner
+/***************************************************************************
+
+    coco_midi.cpp
+
+	Emulation of Rutherford Research's Midi Pak and also compatible with
+	Go4Retro's MIDI Maestro.
+
+***************************************************************************/
+
+#include "emu.h"
+#include "coco_midi.h"
+
+#include "machine/6850acia.h"
+#include "machine/clock.h"
+#include "bus/midi/midi.h"
+
+#define MC6850_TAG "mc6850"
```

Comment:
It’s pointless having this macro when you use the finder to access it everywhere anyway.  The literal tag wouldn’t be repeated even without the macro.